docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -76,6 +81,18 @@ class DebugPage extends React.PureComponent<Props, State> {
       );
     }
   }
+
+  private static copyTextToClipboard(text: string) {
+    const textArea = document.createElement('textarea');
+    textArea.style.opacity = '0.0';
```

Comment:
Not an expert on this topic, but could this be `display = 'none'`?

Reply:
I've tried `display = 'none'` and `visibility = 'hidden'`, neither works well on chrome.
I guess dom's physical existence and visibility affect the behavior, but also I'm not an expert :)